docs/management/additional-packages.md

```diff
@@ -118,6 +122,10 @@ yum install name_of_package --enablerepo=epel,base,updates
 
 And as above make sure no package from the system will get overwritten in the process.
 
+:::warning
+EPEL 7 reached its end of life at the same time as CentOS 7, so installing additional packages from its repositories is even more at your own risk now. Let us know what packages from there you really can't use XCP-ng without, so that we may consider providing them in our supported repositories.
```

Comment:
Maybe adding that supporting those packages mean backporting security fixes when possible?

Reply:
Done